src/DistributedLock.Tests/Tests/ZooKeeper/ZooKeeperPathTest.cs

```diff
@@ -93,9 +93,8 @@ public void TestExposesMinimalApi()
     {
         var publicMembers = typeof(ZooKeeperPath).GetMembers()
             .Where(m => m.DeclaringType == typeof(ZooKeeperPath));
-        CollectionAssert.AreEquivalent(
-            new[] { "ToString", "Equals", "Equals", "GetHashCode", "op_Equality", "op_Inequality", ".ctor" },
+        Assert.That(
             publicMembers.Select(m => m.Name)
-        );
+, Is.EquivalentTo(new[] { "ToString", "Equals", "Equals", "GetHashCode", "op_Equality", "op_Inequality", ".ctor" }));
```

Comment:
Seems like `NUnit.Analyzers` messed up indentation. I can fix this here and report it to them.

Reply:
Fixed and reported